### DISCUSSION TITLE
Problema distanza sensore porta

### DISCUSSION THREAD

**Discussion by polinter** [indomus/forum#315] [GENERALE - Discussioni generiche]
Ho installato HA su Raspberry con chiavetta Conbee II che funge da hub per i sensori sulle porte di tipo xiaomi.

Funziona tutto perfettamente ad eccezione del sensore del garage che risulta essere il più lontano dalla chiavetta Conbee. 

Mi chiedo come posso fare in modo che anche il sensore più lontano possa essere letto dalla chiavetta. A circa 1.5m dal sensore del garage c'è un altro sensore identico che funziona perfettamente, la tecnologia zigbee non dovrebbe fare da ponte tra i dispositivi?

Ho il dubbio che il muro del garage sia troppo spesso e a prescindere dalla distanza non faccia passare il segnale. Esiste una soluzione alternativa per poter monitorare anche le aperture del box?

**Comment by snakuzzo** [comment]
Dovresti provare a piazzare nel mezzo dei device Zigbee alimentati NON a batteria che nel 99% dei casi fanno anche da router Zigbee.

**Reply by polinter** [reply]
Una lampadina zigbee funge da extender solo quando accesa?

**Reply by snakuzzo** [reply]
Che intendi per accesa ? Deve essere alimentata.
Se la colleghi ad un interruttore fisico e poi la spegni con quell'interruttore, di fatto non è più alimentata ed ovviamente non potrà funzionare.

**Reply by nromagna** [reply]
Stesso problema di Polinter. Grazie per suggerienti.
